src/screens/HomeScreen.tsx

```diff
@@ -100,7 +100,11 @@ const HomeScreen = ({navigation}: any) => {
       set_upcoming_movies_list(upcoming);
 
       const nowplaying = await now_playing_movies_fun();
-      set_now_playing_movies_list(nowplaying);
+      set_now_playing_movies_list([
+        {id: 'dummy1'},
```

Comment:
check if this dummy data have a same object structure that of the `nowplaying`, otherwise while defining the type of `now_playing_movies_list` will throw an error.